Volume/Views/MainView/HomeList.swift

```diff
@@ -83,8 +83,10 @@ struct HomeList: View {
     var body: some View {
         NavigationView {
             ScrollView(showsIndicators: false) {
-                LazyVStack {
-                    Header("The Big Read").padding(.bottom, -12)
+                LazyVStack(spacing: 0) {
```

Comment:
I'm not sure, but can `spacing: 20` here replace all the padding you add in this block?

Reply:
I can, but the reason I did it this way was because adding a spacing of 20 would add spacing/excess spacing between elements that don't need it (Spacers, the Big Read section), as well as add extra spacing to elements that have padding from the default .padding().  Adding the spacing would remove the redundancy of .padding(.bottom, 20), but I would still be writing a decent amount of .padding(-some int) to compensate for this 20 pixel spacing.  In my opinion, this is messier and harder to understand than the .padding(.bottom, 20), but I can still change to that if you'd like.

Reply:
Maggie also gave very specific bounds, and having these padding statements makes it very clear exactly how much space there is between elements.

Reply:
Scratch that, I think I'm wrong about the Spacers.